src/arviz_base/io_cmdstanpy.py

```diff
@@ -29,7 +29,7 @@ def __init__(
         observed_data=None,
         constant_data=None,
         predictions_constant_data=None,
-        log_likelihood=None,
+        log_likelihood=False,
```

Comment:
I would default to True for cmdstanpy, False for the others.

In cmdstanpy that means using the log_lik variable if present, so no computation happens. All the others would compute the log likelihood at conversion time which can be expensive